Testscripts/Linux/nvme_blkdiscard.sh

```diff
@@ -46,6 +47,24 @@ Run_Blkdiscard() {
             LogMsg "Unmounted ${namespace}"
         fi
         #Run blkdiscard on partition
+        which blkdiscard
```

Comment:
this will install util-linux from sources if not found already install.
It produces too much overhead, and if you look here that package updates *a lot* of binaries. We don't want to alter the guest distro that much.
https://packages.ubuntu.com/bionic/amd64/util-linux/filelist

I would suggest to just skip this test instead, based on my other PR.

If @juhlee-microsoft still wants to go with this implementation, it needs to be changed to:
1. use install_package util-linux
2. if that fails (package not found or blkdiscard binary not present after install), then go with this manual install.
3. still have a check at the end of this new code to confirm that blkdiscard is present/was installed on the system.